imap/imap_private.h

```diff
@@ -73,10 +73,14 @@ struct Progress;
 #define IMAP_FLAGS_PENDING    (1 << 4)
 
 /* imap_exec flags (see imap_exec) */
-#define IMAP_CMD_FAIL_OK (1 << 0)
-#define IMAP_CMD_PASS    (1 << 1)
-#define IMAP_CMD_QUEUE   (1 << 2)
-#define IMAP_CMD_POLL    (1 << 3)
+#define IMAP_CMD_PASS    (1 << 0)  /**< Run the imap command and all previous commands queued */
+#define IMAP_CMD_QUEUE   (1 << 1)  /**< Queue a command */
+#define IMAP_CMD_POLL    (1 << 2)  /**< Poll the tcp connection before running the imap command */
+
+/* imap_exec flags (see imap_exec) */
+#define IMAP_EXEC_SUCCESS (1 << 0) /**< Imap command executed or queued succeed */
+#define IMAP_EXEC_ERROR   (1 << 1) /**< Imap command failure */
+#define IMAP_EXEC_FATAL   (1 << 2) /**< Imap connection failure */
```

Comment:
Unlike the flags, above, these return codes are mutually-exclusive.
They would probably be better as an `enum`.

Reply:
done